### PR TITLE
[Correct both Menu spacing issue]

### PR DESCRIPTION
**Pull request by hitesh-wagento**
### Description (*)
Fixed both Menu spacing issues in the mobile view.

### Fixed Issues (if relevant)

1. magento/magento2#26235: Both Menu spacing should be same

### Manual testing scenarios (*)

1. Go to the mobile view
2. Click on Humber menu

### Expected result

1. Both Menu spacing should be the same
expected

### Actual result

1. Both Menu spacing different
actual

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by hitesh-wagento** [comment]
@dmytro-ch 

Can you please guide me how can I resolve Functional Tests error.

Thanks

**Comment by dmytro-ch** [comment]
Hi @hitesh-wagento,
the functional test error is not relevant to your changes. Please just merge the latest `2.4-develop` and it will solve the problem.
Thank you!

**Comment by hitesh-wagento** [comment]
Hi @dmytro-ch

Thank you for the reply I have merged 2.4-develop in my branch

Thanks

File: app/design/frontend/Magento/blank/web/css/source/_navigation.less [hunk 1]
```diff
@@ -154,7 +154,7 @@
                 &.greet.welcome {
                     border-top: 1px solid @color-gray82;
                     font-weight: @font-weight__bold;
-                    padding: .8rem @indent__base;
+                    padding: .8rem 15px;
```
Comment (ptylek) [inline app/design/frontend/Magento/blank/web/css/source/_navigation.less]: Please use @submenu__padding-left variable.

File: app/design/frontend/Magento/blank/web/css/source/_navigation.less [hunk 2]
```diff
@@ -168,7 +168,7 @@
                 .lib-css(text-decoration, @navigation-level0-item__text-decoration);
                 display: block;
                 font-weight: @font-weight__bold;
-                padding: .8rem @indent__base;
+                padding: .8rem 15px;
```
Comment (ptylek) [inline app/design/frontend/Magento/blank/web/css/source/_navigation.less]: Please use @submenu__padding-left variable.

**Review by ptylek — CHANGES_REQUESTED** [review]
Changes requested.

@submenu__padding-left variable has value of 15px and can be found in lib/web/css/source/lib/variables/_navigation.less

**Comment by hitesh-wagento** [comment]
Hi @ptylek 

I have done changes as per your requirement

Thanks

**Review by ptylek — CHANGES_REQUESTED** [review]
Hey, @submenu__padding-left is equal to 15px therefore this line and another one should appear as:
`padding: .8rem @submenu__padding-left;`

**Comment by hitesh-wagento** [comment]
Hi @ptylek 

Sorry for the mistake I have again changed and push my changes

Thanks

**Review by ptylek — APPROVED** [review]
Thanks for your contribution!

**Comment by magento-engcom-team** [comment]
Hi @ptylek, thank you for the review.
[ENGCOM-6579](https://jira.corp.magento.com/browse/ENGCOM-6579) has been created to process this Pull Request

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-6579](https://jira.corp.magento.com/browse/ENGCOM-6579) has been created to process this Pull Request

**Comment by engcom-Delta** [comment]
:heavy_check_mark: QA passed

**Pull request merged**